4-projects/business_unit_1/development/example_peering_project.tf

```diff
@@ -68,12 +68,11 @@ module "peering_network" {
 }
 
 module "peering" {
-  source = "terraform-google-modules/network/google//modules/network-peering"
-
-  prefix        = "bu1-d"
-  local_network = module.peering_network.network_self_link
-  peer_network  = data.google_compute_network.shared_vpc.self_link
-
+  source            = "terraform-google-modules/network/google//modules/network-peering"
+  version           = "~> 2.0"
```

Comment:
@bharathkkb should the `peering_network` module also be bumped to `3.x` so that both are in sync? 

```
module "peering_network" {
  source                                 = "terraform-google-modules/network/google"
  version                                = "~> 2.0"
```

Reply:
The peered network is using  `3.x`:

https://github.com/terraform-google-modules/terraform-example-foundation/blob/846f90244744d312fd6b272897f31312006a5022/3-networks/modules/base_shared_vpc/main.tf#L45